src/components/ChatList/ChannelItem.style.ts

```diff
@@ -16,7 +16,7 @@ export const channelItemStyles = StyleSheet.create({
     paddingBottom: dimen.normalizeDimen(14),
     paddingStart: dimen.normalizeDimen(15),
     paddingEnd: dimen.normalizeDimen(20),
-    borderBottomColor: COLORS.lightgrey,
+    borderBottomColor: COLORS.gray200,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color of the border bottom has been changed from `COLORS.lightgrey` to `COLORS.gray200`. Ensure that this color change aligns with the overall design and accessibility guidelines, as it could impact readability and user experience.
```diff
-    borderBottomColor: COLORS.lightgrey,
+    borderBottomColor: COLORS.gray200,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->